packages/features/ee/organizations/lib/orgDomains.ts

```diff
@@ -64,3 +64,8 @@ export function getSlugOrRequestedSlug(slug: string) {
     ],
   } satisfies Prisma.TeamWhereInput;
 }
+
+export function userOrgQuery(hostname: string, fallback?: string | string[]) {
+  const { currentOrgDomain, isValidOrgDomain } = orgDomainConfig(hostname, fallback);
```

Comment:
This is already being called in the parent: https://github.com/calcom/cal.com/pull/11288/files#diff-0a49c6d137ab5a519fee377425ff051a13bcf55ff728bca45daaa73a4c5f0458R144-R147

Can we either just pass the computed data or find another abstraction?

Reply:
I noticed that `orgDomainConfig` and `getSlugOrRequestedSlug` were called together in multiple places throughout the app, so I created this function. Looking through the codebase I noticed this is happening more than just the file that is linked.